README.md

```diff
@@ -10,3 +10,9 @@ Please download, unzip, and copy the corresponding asset folder for your platfor
 * Android - `/android/bin/release/`
 * Win32 - The win32 binaries are not included in the zip assets. They are available on download on [nuget](https://www.nuget.org/packages/Microsoft.PlayFab.PlayFabParty.Cpp.Windows).
 * Linux - `/x64/Release/libparty.so`
+
+## Troubleshooting
+
+### WSL Compatibility
```

Comment:
Shouldn't these sorts of docs be published to the public MSDN docs instead of this README? it's probably fine to have them in both places but the MSDN docs are the more comprehensive source of truth for API considerations like this.

Reply:
They should but we haven't allocated the work for that yet. It is logged. This is just a quick fix to support the customers.